src/components/styles/spacings.js

```diff
@@ -0,0 +1,19 @@
+export default {
```

Comment:
In general, is not a good idea exporting default, what do you think about unifying all these things in an only file and export _spacings_, _fonts_, etc?